src/fragments/lib/in-app-messaging/js/sync-messages.mdx

```diff
@@ -0,0 +1,10 @@
+The first step to getting in-app messages in front of your app’s active users is to make sure all available In-App Messaging campaign messages are synced to their device, which would be triggered when an analytics or In-App Messaging event is triggered. Amplify In-App Messaging empowers you with the control of when and how often you perform this sync.
```

Comment:
Run-on sentence! Suggested:

To trigger messages, you must sync them from your In-App Messaging campaigns to your active user's devices. These messages are then triggered with an analytics or In-App Messaging event. You can control when and how often this sync is performed